src/rebar_prv_install_deps.erl

```diff
@@ -123,22 +126,29 @@ format_error(Reason) ->
 -spec handle_deps(atom(), rebar_state:t(), list()) ->
                          {ok, [rebar_app_info:t()], rebar_state:t()} | {error, string()}.
 handle_deps(Profile, State, Deps) ->
-    handle_deps(Profile, State, Deps, false).
+    handle_deps(Profile, State, Deps, false, []).
 
--spec handle_deps(atom(), rebar_state:t(), list(), boolean() | {true, binary(), integer()})
+-spec handle_deps(atom(), rebar_state:t(), list(), list() | boolean()) ->
+                         {ok, [rebar_app_info:t()], rebar_state:t()} | {error, string()}.
+handle_deps(Profile, State, Deps, Update) when is_boolean(Update) ->
+    handle_deps(Profile, State, Deps, Update, []);
+handle_deps(Profile, State, Deps, Locks) when is_list(Locks) ->
+    handle_deps(Profile, State, Deps, false, Locks).
```

Comment:
Given this is private stuff, can we just skip this clause entirely or something?